docs/TEAMS.md

```diff
@@ -0,0 +1,202 @@
+The Organic Maps is led by contributors from across the community. Different functional domains may periodically establish, develop, or dissolve their own leadership structures, whether formal or informal.
+
+The file contains the functional project structure. The structure of GitHub teams must match the content of this file. Any changes to memberships must be performed via a PR to this file.
+
+- [Android Team](#android-team)
+- [iOS Team](#ios-team)
+- [Map Data Team](#map-data-team)
+- [Map Styles Team](#map-styles-team)
+- [DevOps Team](#devops-team)
+- [Design Team](#design-team)
+- [Website Team](#website-team)
+- [Growth Team](#growth-team)
+- [Translation Team](#translation-team)
+- [User Support Team](#user-support-team)
+- [Legal Team](#legal-team)
```

Comment:
Add Android Auto, CarPlay (I guess it will be the same as iOS right now) and cpp teams.

Reply:
I think its better to make AA a sub-team of Android.

Reply:
cpp aka App Core Team:
rendering, routing, search, editor, bookmarks & tracks, desktop and mobile linux versions, etc.

Reply:
@rtsisyk Releng is meant to be a part of Devops?

What about communication channels (chats, social media, moderation)?
The social media and news could belong to the Growth tem. But chats and moderation?

Reply:
I kept C++ for now. We can add more specific "search", "routing", etc. later.

Reply:
So what about a release management team?